core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/datalake/dr/BackupRestoreStatusService.java

```diff
@@ -61,7 +61,15 @@ public void restoreDatabaseFinished(long stackId) {
 
     public void handleDatabaseRestoreFailure(long stackId, String errorReason, DetailedStackStatus detailedStatus) {
         clusterService.updateClusterStatusByStackId(stackId, Status.RESTORE_FAILED, errorReason);
-        stackUpdater.updateStackStatus(stackId, detailedStatus);
+        stackUpdater.updateStackStatus(stackId, detailedStatus, extractSaltErrorIfAvailable(errorReason));
         flowMessageService.fireEventAndLog(stackId, Status.UPDATE_FAILED.name(), DATALAKE_DATABASE_RESTORE_FAILED, errorReason);
     }
+
+    private String extractSaltErrorIfAvailable(String errorReason) {
+        String errors = "Error(s): ";
```

Comment:
Perhaps worth extracting as a (private) constant. 😄

Reply:
Curse you, you found the one thing where I was like, "Should I make this a constant? Eh, it's not a big deal." :laughing:

I'll fix it in the next patch, but waiting to see if there are any other comments first.

Reply:
Sure, definitely not a burning issue. 😛